### DISCUSSION TITLE
Change Database Tables From Users To Another

### DISCUSSION THREAD

**Discussion by Teddy-time** [orchidsoftware/platform#2203] [Q&A]
Hi,
My Laravel/Orchid platforms are:
Laravel: 8.83.8
Orchid: 11.0.1

Am having an issue changing from tables from the Users table to my own custome Orchid_admin table. I have effected the following changes:
config\auth.php
    'guards' => [
       
        'adminorchid' => [
            'driver' => 'session',
            'provider' => 'adminsorchid',
        ],

        'adminorchid-api' => [
            'driver' => 'token',
            'provider' => 'adminsorchid',
            'hash' => false,
        ],
    ],

    'providers' => [
       
        'adminsorchid' => [
            'driver' => 'eloquent',
            'model' => App\Models\Orchid_admin::class,
        ],

    ],

And in this:
config\platform.php

'guard' => config('auth.defaults.guard', 'adminorchid'),

And over here:
app\Providers\AppServiceProvider.php
Dashboard::useModel(
            \Orchid\Platform\Models\User::class,
            \App\Models\Orchid_admin::class
        );

Yet, orchid is still using the 'Users' database table for authentication.

Any insights would be appreciated

**Comment by tabuna** [comment]
```diff
- 'guard' => config('auth.defaults.guard', 'adminorchid'),
+ 'guard' => config('auth.guards.adminorchid'),
```

You are still using the default value. Need to return your `guards `
